pkg/api/converterfromapi.go

```diff
@@ -599,6 +599,14 @@ func convertWindowsProfileToV20170701(api *WindowsProfile, v20170701Profile *v20
 func convertWindowsProfileToVLabs(api *WindowsProfile, vlabsProfile *vlabs.WindowsProfile) {
 	vlabsProfile.AdminUsername = api.AdminUsername
 	vlabsProfile.AdminPassword = api.AdminPassword
+	if api.ImageRef != nil {
+		vlabsProfile.ImageRef = &vlabs.ImageReference{}
+		vlabsProfile.ImageRef.Gallery = api.ImageRef.Gallery
+		vlabsProfile.ImageRef.Name = api.ImageRef.Name
+		vlabsProfile.ImageRef.ResourceGroup = api.ImageRef.ResourceGroup
+		vlabsProfile.ImageRef.SubscriptionID = api.ImageRef.SubscriptionID
+		vlabsProfile.ImageRef.Version = api.ImageRef.Version
+	}
```

Comment:
if this is showing up as negative coverage, it might need one of the test cases that compares the final arm template against a known string

Reply:
Added some unit tests around convert to/from